ide/MoonshineSharedCore/src/components/renderers/ProjectListItemRenderer.mxml

```diff
@@ -7,6 +7,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 // 
+// http://www.apache.org/licenses/LICENSE-2.0 

```

Comment:
@Aszusz I believe that link is incorrect.

Reply:
OK, I got it. You were right about the file being committed twice.

In this commit:
https://github.com/Moonshine-IDE/Moonshine-IDE/commit/8036227f88446c7046e4c19a7e988ca6a1ac3340
I corrected inconsistencies between existing licenses (15 files total), so they can be lexed correctly by regex.

I need to ask you to look at the most recent commits only. If this is too difficult for such a large commit I can alternatively squash all my changes to one commit.